schema-engine/sql-introspection-tests/tests/simple/cockroach/foreign_keys_duplicates_should_be_ignored_23_1_and_up.sql

```diff
@@ -0,0 +1,41 @@
+-- tags=CockroachDb231
```

Comment:
I wanted to use 
```
-- tags=cockroachdb
-- exclude=CockroachDb222,CockroachDb221
```
But that didn't work, if someone is interested to fix that, that could be done outside of this PR